### ISSUE TITLE
Package broken on julia 1.9 Windows

### ISSUE THREAD

**Issue by jebej** [DynareJulia/FastLapackInterface.jl#31]
The package does not work on julia 1.9 due to a `libblastrampoline` issue:
```
Testing Running tests...
gglse!: Error During Test at C:\Users\jebej\.julia\packages\FastLapackInterface\7rWJc\test\lse_test.jl:16
  Got exception outside of a @test
  could not load library "libblastrampoline"
  The specified module could not be found.
  Stacktrace:
    [1] gglse!(ws::LSEWs{Float32}, A::Matrix{Float32}, c::Vector{Float32}, B::Matrix{Float32}, d::Vector{Float32}; resize::Bool, blocksize::Int64)
      @ FastLapackInterface C:\Users\jebej\.julia\packages\FastLapackInterface\7rWJc\src\lse.jl:101
    [2] gglse!(ws::LSEWs{Float32}, A::Matrix{Float32}, c::Vector{Float32}, B::Matrix{Float32}, d::Vector{Float32})
      @ FastLapackInterface C:\Users\jebej\.julia\packages\FastLapackInterface\7rWJc\src\lse.jl:69
```

```
julia> versioninfo()
Julia Version 1.9.0-beta4
Commit b75ddb787f (2023-02-07 21:53 UTC)
Platform Info:
  OS: Windows (x86_64-w64-mingw32)
  CPU: 4 × Intel(R) Core(TM) i7-3770K CPU @ 3.50GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.6 (ORCJIT, ivybridge)
  Threads: 4 on 4 virtual cores
Environment:
  JULIA_CUDA_USE_BINARYBUILDER = false
  JULIA_NUM_THREADS = 4
```

**Comment by MichelJuillard** [comment]
I'm not able to reproduce the problem when running julia-1.9.0-beta4 on a Linux/Debian installation.
It looks more like an installation problem of julia-1.9.0-beta4 under Windows
Can you please give the response to
```
julia> ]
pkg> status --manifest
```

**Comment by jebej** [comment]
Is there a reason why this:
https://github.com/DynareJulia/FastLapackInterface.jl/blob/721e98f67ab17c69b37020e998a77288dc0ba50f/src/FastLapackInterface.jl#L12-L16
isn't only:
```
using LinearAlgebra.LAPACK: liblapack
```

**Comment by jebej** [comment]
Yes this is a windows issue, I dug a bit and found: https://github.com/JuliaLang/julia/issues/48427

**Comment by jebej** [comment]
This is what I get

```
julia> LinearAlgebra.LAPACK.liblapack
"libblastrampoline-5.dll" # julia1.9 beta 4

julia> LinearAlgebra.LAPACK.liblapack
"libblastrampoline" # julia 1.8.5

julia> LinearAlgebra.LAPACK.liblapack
"libopenblas64_" # julia 1.6.7
```

**Comment by jebej** [comment]
Fixed on master

**Comment by jebej** [comment]
@MichelJuillard any chance of getting a release soon to solve the issue on 1.9 on Windows?

**Comment by MichelJuillard** [comment]
@jebej I just released version v1.2.9
